src/mpyl/steps/deploy/bpm_deploy.py

```diff
@@ -20,9 +20,6 @@ def __init__(self, logger: Logger) -> None:
                 version="0.0.1",
                 stage=STAGE_NAME,
             ),
-            produced_artifact=ArtifactType.NONE,
-            # TO DO: create bpm artifact:
-            # https://vandebron.atlassian.net/browse/BPMN-293
```

Comment:
did you also close the ticket for them?

Reply:
No, because they might need to replace this with something else (I don't know anything about how BPM deploys).